src/Utils/Transport.php

```diff
@@ -44,4 +44,24 @@ public static function parseTransport(?string $transport): int
 
         throw new \Exception("Transport $transport not supported");
     }
+
+    public static function isRestOnly(int $transport): bool
+    {
+        return Transport::compareTransports(Transport::REST, $transport);
+    }
+
+    public static function isGRPCOnly(int $transport): bool
+    {
+        return Transport::compareTransports(Transport::GRPC, $transport);
+    }
+
+    public static function isGRPCRest(int $transport): bool
```

Comment:
nit, but convention is to camel-case gRPC as `Grpc` in class and method names

```suggestion
    public static function isGrpcRest(int $transport): bool
```